lib/chef-dk/command/verify.rb

```diff
@@ -141,6 +147,35 @@ def components
           # ------------
           failures = []
 
+          # fail the verify if we have more than one version of chef-provisioning or any of its drivers.
```

Comment:
@ksubrama @jkeiser @danielsdeleo Y'all on board with having version-uniqueness be a hard error for `chef-provisioning` and its drivers? Looks like there's one :+1: and one :-1: for that.